src/App.js

```diff
@@ -1,11 +1,22 @@
 import "./App.css"
 import { Routes, Route } from "react-router-dom"
 import { Login } from "./components/auth/Login"
+import { Authorized } from "./views/Authorized"
+import { ApplicationViews } from "./views/ApplicationViews"
+import { Navbar } from "./navbar/Navbar"
 
 export const App = () => {
   return (
+    <>
+    <Navbar />
     <Routes>
       <Route path="/login" element={<Login />} />
-    </Routes>
-  )
-}
+
+      <Route path="/" element={
```

Comment:
Replace path="/" with path="*" to route ANY url other than /login (and later, /register) through the Authorized component.

Reply:
After we talked through it, I thought it would be fine to leave the path as "/". However when I tested this again, I noticed the following error in the console related to this line: 
"You rendered descendant <Routes> (or called `useRoutes()`) at "/" (under <Route path="/">) but the parent route path has no trailing "*". This means if you navigate deeper, the parent won't match anymore and therefore the child routes will never render.

Please change the parent <Route path="/"> to <Route path="*">."

Following the console's suggestion of changing the path to "*" fixed the problem. I say let's go ahead and fix this issue now before merging.